content/en/docs/setup/production-environment/tools/kubeadm/high-availability.md

```diff
@@ -226,15 +226,9 @@ option. Your cluster requirements may need a different configuration.
    As stated in the command output, the certificate key gives access to cluster sensitive data, keep it secret!
    {{< /caution >}}
 
-1. Apply the CNI plugin of your choice:  
-   [Follow these instructions](/docs/setup/production-environment/tools/kubeadm/create-cluster-kubeadm/#pod-network) 
-   to install the CNI provider. Make sure the configuration corresponds to the Pod CIDR specified in the
-   kubeadm configuration file (if applicable).
-
-   {{< note >}}
-   You must pick a network plugin that suits your use case and deploy it before you move on to next step.
-   If you don't do this, you will not be able to launch your cluster properly.
-   {{< /note >}}
+    {{< note >}}
+    You must pick a network plugin that suits your use case and deploy it before you move on to next step. If you don't do this, your cluster will not launch properly.
```

Comment:
Should the change to  content/en/../../high-availability.md be made in a separate PR? That change is not related to the localization effort.
This PR's description links it to the hindi localization tracking issue.
Open to suggestions.